product_barcode_constraint_per_company/__manifest__.py

```diff
@@ -1,18 +1,17 @@
-# Copyright (C) 2020-Today: GRAP (http://www.grap.coop)
-# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# Copyright 2023 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
     "name": "Product Barcode Constraint per Company",
     "version": "14.0.1.0.1",
     "category": "Product",
-    "summary": "Change the product barcode constraint"
-    ", allowing the same barcode for differents companies",
-    "author": "GRAP, Odoo Community Association (OCA)",
```

Comment:
could you preserve authorship, and add Cetmix instead of replacing ? thanks !

Reply:
okay, thanks